src/content/docs/logs/logs-context/net-configure-logs-context-all.mdx

```diff
@@ -24,11 +24,125 @@ redirects:
+
+## Automatic logs in context options [#automatic]
+
+You have three options to configure APM logs in context to send your app's logs and linking metadata automatically to New Relic.
+
+<CollapserGroup>
+
+  <Collapser
+    id="1-agent"
+    title="Option 1 (simplest): Let the agent decorate and forward your logs."
+  >
+
+  This is the simplest approach, and it's a great choice for developers who may not have the access or interest in setting up a log forwarder, or for accounts that want to see the power of logs and other linking metadata in context of their apps, without a lot of overhead.
+  
+  Using this option, your logs will include `span.id`, `trace.id`, `hostname`, `entity.guid`, and `entity.name`. These attributes automatically link your logs to spans, traces, and other telemetry, making it easier to troubleshoot.
+
+  All you need to do is install an agent version with log forwarding capabilities ([.NET agent 9.7.0 or higher](/docs/release-notes/agent-release-notes/net-release-notes)), and update your configuration to enable forwarding.
+
+  **INSERT .NET-specific configuration**
+
+  **This PR https://github.com/newrelic/docs-website/pull/6767 has some config info.**
+
+  This option may be on by default in a future agent version.
+  
+  **Optional adjustments:**
+
+  Once this is enabled, you also have control over the maximum number of logs sent to New Relic every minute. The default value is 10,000. If more than 10,000 logs are received in a 60-second window, your logs will begin to be sampled.
+  
+  Set it to a higher number to receive more logs. Set it to a lower number to receive fewer logs. Any integer is valid.
+
+  **INSERT .NET-specific configuration**
+
+  If you have an existing log forwarding solution and are updating your agent to use automatic logs in context, be sure to **disable your old log forwarder**. Otherwise, your app will be sending double log lines. Depending on your account, this could result in double billing. For more information, follow the procedures to disable your [specific log forwarder](/docs/logs/forward-logs/enable-log-management-new-relic#log-forwarding).
+  
+  </Collapser>
+
+  <Collapser
+    id="2-decorate"
+    title="Option 2: Let the agent decorate your logs."
+  >
+
+  Already have a log forwarder you like? We've got you covered! Language agents can decorate your logs with the linking metadata needed to provide access to automatic logs-in-context features.
+
+  1. If you are currently sending logs using the old logs-in-context method, be sure to remove that code from your [specific log forwarder](/docs/logs/forward-logs/enable-log-management-new-relic#log-forwarding) before enabling local decoration. Otherwise, your app may send double log lines. Depending on your account, this could result in double billing.
+
+  2. Enable log decorating in your configuration, then relaunch the agent to start decorating your logs.
+  
+  
+
+  **INSERT .NET-specific configuration**
```

Comment:
This block shows how to enable local decorating, but actually getting the logs decorated requires manual changes to the customers config framework.  Do we want to link the .NET specific docs for the specifics or put all that in here?

```xml
<applicationLogging enabled="true">
 <localDecorating enabled="true" />
</applicationLogging>
```

Reply:
I've reworded this to follow how Ruby handled this. OK by you?